### PR TITLE
Update zabbix.yml

### PR DESCRIPTION
**Pull request by Solaris17**
Zabbix repo does not contain ubuntu focal on zabbix version 4.2: https://repo.zabbix.com/zabbix/4.2/ubuntu/dists/

##### SUMMARY
Zabbix repo does not contain ubuntu focal on zabbix version 4.2: https://repo.zabbix.com/zabbix/4.2/ubuntu/dists/

Deployments to focal (20.04) specifying 4.2 will fail.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
Affects the "zabbix_version:" variable.

### CODE REVIEW

**Comment by D3DeFi** [comment]
@Solaris17 can you please rebase this on latest changes in `main` and resubmit? We had some problems with CI that should be now addressed with #374, thanks!

**Comment by D3DeFi** [comment]
@Solaris17 if you need some additional context for rebase please look at this url:
https://docs.ansible.com/ansible/latest/dev_guide/developing_rebasing.html

tl;dr version:
```bash
git remote add upstream https://github.com/ansible-collections/community.zabbix.git  # if not done already
git fetch upstream
git checkout patch-1
git rebase upstream/main
```

**Comment by Solaris17** [comment]
@D3DeFi Hey there, sorry. I've been on vacation I will get to this as soon as I can.
